lib/plugins/graphqlBranches.js

```diff
@@ -86,7 +86,7 @@ const blacklistBranchProtectionAttributes = (mapdata) =>
     .filter((f) => !blacklistedAttributes.includes(f))
     .reduce((a, k) => ({ ...a, [k]: mapdata[k] }), {});
 
-module.exports = class Branches2 {
+module.exports = class graphqlBranches {
```

Comment:
please uppercase exported classes for better readability. Just a best practice.
```suggestion
module.exports = class GraphqlBranches {
```